src/test/java/com/conveyal/datatools/manager/jobs/GisExportJobTest.java

```diff
@@ -0,0 +1,230 @@
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import static com.conveyal.datatools.TestUtils.createFeedVersion;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class GisExportJobTest {
+    private static final Logger LOG = LoggerFactory.getLogger(GisExportJobTest.class);
+    private static Project project;
+    private static FeedVersion calTrainVersion;
+    private static FeedVersion hawaiiVersion;
+
+    @BeforeClass
+    public static void setUp() {
+        DatatoolsTest.setUp();
+        LOG.info("ProcessGtfsSnapshotMergeTest setup");
+
+        // Create a project, feed sources, and feed versions to merge.
+        project = new Project();
+        project.name = String.format("Test %s", new Date().toString());
+        Persistence.projects.create(project);
+        FeedSource caltrain = new FeedSource("Caltrain");
+        Persistence.feedSources.create(caltrain);
+        calTrainVersion = createFeedVersion(caltrain, "caltrain_gtfs.zip");
+        FeedSource hawaii = new FeedSource("Hawaii");
+        Persistence.feedSources.create(hawaii);
+        hawaiiVersion = createFeedVersion(hawaii, "hawaii_fake_no_shapes.zip");
+    }
+
+    @Test
+    public void canExportStops () throws IOException {
+        // Run the GIS export job for stops.
+        File zipFile = File.createTempFile("stops", ".zip");
+        Set<String> ids = new HashSet<>();
+        ids.add(calTrainVersion.id);
+        GisExportJob gisExportJob = new GisExportJob(GisExportJob.ExportType.STOPS, zipFile, ids, "test");
+        gisExportJob.run();
+        assertThat(gisExportJob.status.error, equalTo(false));
+        FeatureCollection collection = getFeatureCollectionFromZippedShapefile(zipFile);
+        // Iterate over features.
+        int featureCount = 0;
+        try (FeatureIterator iterator = collection.features()) {
+            while (iterator.hasNext()) {
+                featureCount++;
+                Feature feature = iterator.next();
+                // GeometryAttribute sourceGeometry = feature.getDefaultGeometryProperty();
+                Collection<Property> properties = feature.getProperties();
+                // Iterate over feature properties and verify everything looks OK.
+                for (Property property : properties) {
+                    String name = property.getName().toString();
+                    LOG.info("{}: {}", name, property.getValue());
+                    if ("the_geom".equals(name)) {
+                        // Check that the geometry was exported properly.
+                        assertThat(property.getValue(), notNullValue());
+                    }
+                }
+            }
+        }
+        // Ensure that all stops from feed version are present in shapefile.
+        assertThat(featureCount, equalTo(calTrainVersion.feedLoadResult.stops.rowCount));
```

Comment:
Can you add one more assertion about whether the correct coordinates of a feature have been returned? It's good to have some baselines of whether geometry is not null and whether the count matches, but we should also make sure some data we're getting back matches expectations.